docs/plugins/structure-of-a-plugin.md

```diff
@@ -57,13 +57,14 @@ In the `src` folder we get to the interesting bits. Check out the `plugin.ts`:
 
 ```jsx
 import {
-  createPlugin,
-  createRoutableExtension,
-} from '@backstage/core-plugin-api';
+  createFrontendPlugin,
+  FrontendPlugin,
+} from '@backstage/frontend-plugin-api';
+import { createRoutableExtension } from '@backstage/core-plugin-api';
 
 import { rootRouteRef } from './routes';
 
-export const examplePlugin = createPlugin({
+export const examplePlugin = createFrontendPlugin({
```

Comment:
Can you revert the changes in this file, please? What was here previously was correct, this is for the current frontend system, anything from the `@backstage/frontend-plugin-api` package is intended to be for the New Frontend System 👍

Reply:
This was definitely a mistake on my part :)